push-messaging-and-notifications/manifest.sample.json

```diff
@@ -7,6 +7,6 @@
       }],
   "start_url": "./index.html?homescreen=1",
   "display": "standalone",
-  "gcm_sender_id": "<YOUR PROJECT NUMBER>",
+  "gcm_sender_id": "<Your Project Number from https://console.developers.google.com>",
   "gcm_user_visible_only": true
```

Comment:
although we can't really include comments in the JSON, we can include one in the README to add more clarity.